R/course.R

```diff
@@ -273,6 +295,25 @@ tidy_unzip <- function(zipfile, cleanup = FALSE) {
   invisible(target)
 }
 
+## takes a Dropbox or Google Drive URL as provided in the web browser, and
+## converts into an URL suitable for direct download (as expected by
+## tidy_download() for instance).
+convert_teaching_url <- function(url) {
+  stopifnot(is.character(url))
```

Comment:
If you're really expecting a character of length one, we have `is_string()` for this.